[2023-10-22T01:13:10.890+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dagg.scrape_data manual__2023-10-22T01:13:10.350350+00:00 [queued]>
[2023-10-22T01:13:10.900+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dagg.scrape_data manual__2023-10-22T01:13:10.350350+00:00 [queued]>
[2023-10-22T01:13:10.901+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-22T01:13:10.922+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): scrape_data> on 2023-10-22 01:13:10.350350+00:00
[2023-10-22T01:13:10.927+0000] {standard_task_runner.py:57} INFO - Started process 642 to run task
[2023-10-22T01:13:10.929+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'first_dagg', 'scrape_data', 'manual__2023-10-22T01:13:10.350350+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/first_dags.py', '--cfg-path', '/tmp/tmp2ev4e7pm']
[2023-10-22T01:13:10.932+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask scrape_data
[2023-10-22T01:13:10.977+0000] {task_command.py:416} INFO - Running <TaskInstance: first_dagg.scrape_data manual__2023-10-22T01:13:10.350350+00:00 [running]> on host 88ab045975f5
[2023-10-22T01:13:11.065+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='me' AIRFLOW_CTX_DAG_ID='first_dagg' AIRFLOW_CTX_TASK_ID='scrape_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T01:13:10.350350+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-22T01:13:10.350350+00:00'
[2023-10-22T01:13:11.066+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-22T01:13:11.066+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /home/vertikal/data_science/web_scrap/rumah/rumah/ > mypipe1 && ./script2.sh > mypipe1']
[2023-10-22T01:13:11.075+0000] {subprocess.py:86} INFO - Output:
[2023-10-22T01:13:11.077+0000] {subprocess.py:93} INFO - /bin/bash: line 1: cd: /home/vertikal/data_science/web_scrap/rumah/rumah/: No such file or directory
[2023-10-22T01:13:11.077+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-10-22T01:13:11.088+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-10-22T01:13:11.092+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=first_dagg, task_id=scrape_data, execution_date=20231022T011310, start_date=20231022T011310, end_date=20231022T011311
[2023-10-22T01:13:11.108+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 14 for task scrape_data (Bash command failed. The command returned a non-zero exit code 1.; 642)
[2023-10-22T01:13:11.142+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-22T01:13:11.158+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
